plugins/Profiler/views/NoInteractionsMessage.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+'use strict';
+
+import React from 'react';
+import {sansSerif} from '../../../frontend/Themes/Fonts';
+
+type Props = {|
+  hasMultipleRoots: boolean,
+  height: number,
+  width: number,
+|};
+
+export default ({ hasMultipleRoots, height, width }: Props) => {
+  let headerText;
+  if (hasMultipleRoots) {
+    headerText = 'No interactions were recorded for the selected root.';
+  } else {
+    headerText = 'No interactions were recorded.';
+  }
+
+  return (
+    <div style={{
+      height,
+      width,
+      flex: 1,
+      display: 'flex',
+      flexDirection: 'column',
+      alignItems: 'center',
+      justifyContent: 'center',
+    }}>
+      <p style={{
+        fontSize: sansSerif.sizes.large,
+      }}>
+        {headerText}
+      </p>
+      {hasMultipleRoots && (
+        <p>
+          You may want to select a different root in the <strong>Elements</strong> panel.
+        </p>
+      )}
+      <p>
+        <a href="https://reactjs.org/docs/interaction-tracking.html" target="_blank">
```

Comment:
Can we make this an fb.me URL and then add a todo list to point it to the right thing together with the release that includes it?

Reply:
Sure. Done.

Although I think fb.me URLs don't support being edited after they're created– so I'm not sure how much value this actually provides vs a direct reactjs.org link.

Reply:
They do support being edited.

Reply:
I thought they used to! But the last two times I've looked for a way to edit one, it doesn't appear to exist anymore, and the tool specifically says that you shouldn't edit them because they might be cached externally. Maybe this changed recently?

Reply:
Or maybe I just overlooked the dialog 😄 Hopefully that's all!

Reply:
It's weird because there are two tools.

Go to the one that says "avoid" and then add `/create/` to its URL. You'll find the one that allows it (after clicking okay in a confirmation dialog).

Reply:
Oh nice! Thanks